content/docs/examples.md

```diff
@@ -476,7 +476,7 @@ $ task all
 Do I have the latest released version?
 
 ```
-$ curl https://gothenburgbitfactory.org/latest/task
+$ curl -s https://taskwarrior.org/download/|grep -Eo 'v[0-9].{1,4}'
```

Comment:
It may return the right result, but parses a webpage made for humans, i.e. it searches for the needle in the haystack. The right thing here would be to fix the URL above or use the non-redirected one...